suites/pacific/rbd/tier-2_rbd_regression.yaml

```diff
@@ -128,6 +128,12 @@ tests:
       name: Test snap and clone creation on restored image from trash
       polarion-id: CEPH-11413
 
+  - test:
+      desc: verify for parent image deletion after flattening the clone and removing snap
+      module: rbd_clone_delete_parent_image.py
+      name: verify for parent image deletion after flattening the clone and removing snap
```

Comment:
In test `name` Instead of `Verify` can we make it `Test for` To keep same format with other test?

Reply:
Updated & done